src/Components/ADT3DSceneBuilder/Internal/Behaviors/Internal/WidgetsTab.tsx

```diff
@@ -147,12 +147,13 @@ const WidgetsTab: React.FC = () => {
     const commonPanelStyles = getLeftPanelStyles(theme);
     const actionButtonStyles = getActionButtonStyles(theme);
     return (
-        <>
-            <Stack tokens={{ childrenGap: 8 }}>
-                <Text className={commonPanelStyles.text}>
-                    {t(LOC_KEYS.tabDescription)}
-                </Text>
-                <div className={commonPanelStyles.formTabContents}>
+        <div className={commonPanelStyles.formTabContents}>
+            <div className={commonPanelStyles.paddedLeftPanelBlock}>
```

Comment:
can we move teh padding to the stack here too? I don't think we really need the padding around the dialog right?